custom_components/better_thermostat/climate.py

```diff
@@ -635,17 +645,17 @@ async def async_set_temperature(self, **kwargs):
 	@property
 	def min_temp(self):
 		"""Return the minimum temperature."""
-		if self._min_temp is not None:
-			return self._min_temp
+		if self._TRV_min_temp is not None:
+			return self._TRV_min_temp
```

Comment:
What's the intention here? Isn't min_temp for returning what we can offer to HA as temperature setpoints? 

So we should return the _min_temp instead of what the TRV could do.

Reply:
no the TRV needs min max, so if nothing is set in the config we use the min max values provided by the TRV itself. but when a user is set min max in the config we use this values